ui/pages/prelogin/index.js

```diff
@@ -0,0 +1,42 @@
+import LoginLayout from '../../components/layouts/login'
+import Cookies from 'universal-cookie'
+
+export default function PreLogin() {
+  const cookies = new Cookies()
+  const organizations = cookies.get('orgs')
+  console.log(organizations)
+
+  return (
+    <>
+      <h1 className='text-base font-bold leading-snug'>Welcome to Infra</h1>
+      <h2 className='my-1.5 mb-4 max-w-md text-center text-xs text-gray-400'>
+        Choose your organization to login to.
+      </h2>
+      {organizations?.length > 0 && (
+        <>
+          <Organizations organizations={organizations} />
+        </>
+      )}
+    </>
+  )
+}
+
+function Organizations({ organizations }) {
```

Comment:
Better to keep this in the above component since it isn't re-used anywhere.

Reply:
done, see https://github.com/infrahq/infra/pull/3142/checks